src/page-multi-facility/CreateBaselineScenarioPage.tsx

```diff
@@ -47,6 +47,7 @@ const CreateBaselineScenarioPage: React.FC = () => {
         dataSharing: true,
         dailyReports: true,
         addFacilities: true,
+        newModelInputs: false,
```

Comment:
Not sure I immediately follow: why is this one set to false?

Reply:
Essentially, we only want this promo to be shown for existing scenarios (newly created ones should already be using recovered cases/deaths), so we set it to false here. Existing scenarios won't have this promo status yet, so we set it to true in `src/database/type-transforms.tsx` only if it is undefined, if that makes sense